schemas/json/theme.json

```diff
@@ -781,258 +780,269 @@
-				},
-				"core/preformatted": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/pullquote": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/query": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/query-pagination": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/query-pagination-next": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/query-pagination-numbers": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/query-pagination-previous": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/query-title": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/quote": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/rss": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/search": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/separator": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/shortcode": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/site-logo": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/site-tagline": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/site-title": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/social-link": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/social-links": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/spacer": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/table": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/table-of-contents": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/tag-cloud": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/template-part": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/term-description": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/text-columns": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/verse": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/video": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/widget-area": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
-				},
-				"core/legacy-widget": {
-					"$ref": "#/definitions/stylesPropertiesAndElementsComplete"
+			"allOf": [
```

Comment:
Is `allOf` required here, or can `paternProperties` just be used alongside `properties`?

Reply:
Yes. I that should work. I followed how it was done in `settings.blocks`.

I'll update the PR and change it in `settings.blocks` as well (for consistency).

Reply:
@ajlende Updated.

Testet locally that:
- autosuggest for core blocks still works
- custom blocks are allowed
- custom properties with invalid name shows warning